frontend/src/routes/(authenticated)/org/[org_id]/+page.svelte

```diff
@@ -0,0 +1,154 @@
+  async function openChangeMemberRoleModal(member: OrgUser): Promise<void> {
+    await changeMemberRoleModal.open({
+      userId: member.user.id,
+      name: member.user.name,
+      role: member.role
+    });
+  }
+
+  let deleteOrgModal: ConfirmDeleteModal;
+  async function confirmDeleteOrg(): Promise<void> {
+    const result = await deleteOrgModal.open(org.name, async () => {
+      const { error } = await _deleteOrg(org.id);
+      return error?.message;
+    });
+    if (result.response === DialogResponse.Submit) {
+      notifyWarning($t('org_page.notifications.delete_org', { name: org.name }));
+      await goto('/');
+    }
+  }
+
+  async function leaveOrg(): Promise<void> {
+    const result = await _deleteOrgUser(org.id, user.id);
+    if (result.error) {
+      notifyWarning($t(`org_page.notifications.leave_org_error`, { name: org.name }));
+    } else {
+      notifySuccess($t(`org_page.notifications.leave_org`, { name: org.name }));
+      await goto('/');
+    }
+  }
+</script>
+
+<DetailsPage wide title={org.name}>
+  <svelte:fragment slot="actions">
+    {#if canManage}
+      <Button variant="btn-success"
+        on:click={openAddOrgMemberModal}>
+        <span class="admin-tabs:hidden">
+          {$t('org_page.add_user.add_button')}
+        </span>
+        <span class="i-mdi-account-plus-outline text-2xl" />
+      </Button>
+      <AddOrgMemberModal bind:this={addOrgMemberModal} orgId={org.id} />
+    {/if}
+  </svelte:fragment>
+  <div slot="title" class="max-w-full flex items-baseline flex-wrap">
+    <span class="mr-2">{$t('org_page.organization')}:</span>
+    <span class="text-primary max-w-full">
+      <EditableText
+        disabled={!canManage}
+        value={org.name}
+        validation={orgNameValidation}
+        saveHandler={updateOrgName}
+      />
+    </span>
+  </div>
+  <div class="mt-6">
+    <!-- TODO: Add project count once orgs can own projects -->
+    <OrgTabs bind:activeTab={$queryParamValues.tab} memberCount={org.members.length} projectCount={0} />
+  </div>
+  <div class="py-6 px-2">
+    {#if $queryParamValues.tab === 'projects'}
+    Projects list will go here once orgs have projects associated with them
+    {:else if $queryParamValues.tab === 'members'}
+    <OrgMemberTable
+      shownUsers={org.members}
+      on:openUserModal={(event) => openUserModal(event.detail)}
+      on:removeMember={(event) => _deleteOrgUser(org.id, event.detail.id)}
+      on:changeMemberRole={(event) => openChangeMemberRoleModal(event.detail)}
+    />
+    {:else if $queryParamValues.tab === 'history'}
+      <div class="space-y-2">
+        <DetailItem title={$t('org_page.details.created_at')} text={$date(org.createdDate)} />
+        <DetailItem title={$t('org_page.details.updated_at')} text={$date(org.updatedDate)} />
+      </div>
+    {:else if $queryParamValues.tab === 'settings'}
+      <div class="flex justify-end">
+        <Button outline variant="btn-error" on:click={leaveOrg}>
```

Comment:
this button should only show if you're a member of the org. If I login as a user who's not in this org and access the org page (which is allowed) then I can't actually leave the org if I'm not a member. Maybe we should just hide the settings tab if the user isn't a member and not an admin.

Reply:
Yeah, a non-member shouldn't even see the Settings tab if there's nothing they can do there. No point in wasting screen real estate.

Reply:
A super-admin should see the tab and be able to delete the org.

Wait, what? I don't think you should be allowed to access the org page or query an org that you're not a member of.
Then everyone would be able to see everyone's names and projects. Super-admins, don't have to be members of course.
But I've only just realized that none of our org GQL queries have any permissions checks on them. 

1) I think you should only be able to query orgById if you're a member of the org.
2) And we need to lock down what fields non-members can query, so that they can see the list of orgs. We could do that by returning a DtoOrg with limited fields or...

We could accomplish both requirements at once, by doing something like: `descriptor.Field(org => org.Members).OrgMembershipRequired();`

Reply:
good thinking ya'll !  Do you think these things should hold up the PR or just be deferred to a new issue?  I'm afraid of this PR sliding into its fifth week...

Reply:
[Beat you](https://github.com/sillsdev/languageforge-lexbox/pull/800#issuecomment-2164820236) 😆 👏

Reply:
Will do this as part of #881.